packages/starrating/src/react/star.js

```diff
@@ -145,12 +146,13 @@ Star.propTypes = {
   onClick: PropTypes.func,
   onEnter: PropTypes.func,
   onLeave: PropTypes.func,
+  size: PropTypes.oneOf(Object.values(Icon.sizes)).isRequired,
```

Comment:
So we have variable sizes of starrating now?  Is that a recent decision?

Reply:
we don't. i added the proptype because it didn't catch the undefined error before. `Icon.sizes.xsmall ` didn't exist. i can move size out of a proptype to make this easier to understand